services/leafSpaceIntegration/leafspaceconnection/main.go

```diff
@@ -0,0 +1,19 @@
+package leafspaceconnection
+
+import (
+	"fmt"
+	"github.com/leanspace/terraform-provider-leanspace/provider"
+)
+
+var LeafSpaceIntegrationDataType = provider.DataSourceType[LeafSpaceConnection, *LeafSpaceConnection]{
+	ResourceIdentifier: "leanspace_leaf_space_integration",
+	Path:               "integration-leafspace/connections",
+	Schema:             leafSpaceConnectionSchema,
+	FilterSchema:       nil,
```

Comment:
There's no find endpoint so did you test this?

Reply:
I confirm there is just a get endpoint to retrieve the current connection (as we can have just one connection per tenant for the moment)

Reply:
I did not get any problem when creating taht why we have ReadPath